cfgov/jinja2/v1/_includes/related-content.html

```diff
@@ -24,7 +24,7 @@
 <div class="related-links">
     <h2 class="header-slug">
         <span class="header-slug_inner">
-            Related links
+            Related content
```

Comment:
The module can be used to deliver more than links, which I think is why the issue was created.  
Happy to revert template name. Any other votes?